connector/src/main/java/org/geysermc/connector/entity/living/merchant/VillagerEntity.java

```diff
@@ -84,4 +92,60 @@ public void updateBedrockMetadata(EntityMetadata entityMetadata, GeyserSession s
         }
         super.updateBedrockMetadata(entityMetadata, session);
     }
+    @Override
```

Comment:
Small nitpick - add a space before this `@Override`.

Reply:
This looks fine to me?

Reply:
Updated - I mean they should return and then define the new function, e.g. 
```
}

@Override
```

Reply:
Gotcha. Manually resolved 🙂